Projects/App/Project.swift

```diff
@@ -21,7 +21,7 @@ let project = Project.makeAppModule(
     .Presentation(implements: .Presentation)
   ],
   sources: ["Sources/**"],
-  resources: ["Resources/**"],
+  resources: ["Resources/**", "Resources/PrivacyInfo.xcprivacy"],
```

Comment:
```suggestion
  resources: ["Resources/**"],
```
`Resources/**` 만 설정해두더라도 같이 적용돼서 없어도 될 것 같아요!